### PR TITLE
Add the support of dm-verity.

### PR DESCRIPTION
**Pull request by tanminger**

### CODE REVIEW

**Comment by kalyankondapally** [comment]
@sgnanase Can u check SKL is not broken :)

**Comment by tanminger** [comment]
I have not SKL device. I can not check it.

File: groups/android_ia/default/init.rc [hunk 1]
```diff
@@ -3,7 +3,6 @@ on fs
 
 on init
     rm /dev/.coldboot_done
-    restart ueventd
```
Comment (shobhitka) [inline groups/android_ia/default/init.rc]: The removing of /dev/.coldboot_done should result in regenerating all the events that were received till now on restarting ueventd. I was not expecting any event to still get lost.

File: groups/android_ia/default/init.rc [hunk 2]
```diff
@@ -83,12 +82,3 @@ service bugreport /system/bin/dumpstate -d -p -B \
...
 
-on init
-   # Load persistent dm-verity state and detect if a restart was
-   # triggered after dm-verity detected a corrupted block
-   verity_load_state
-
-on fs
-   # Update dm-verity persistent state and set partition.*.verified
-   # properties
-   verity_update_state
```
Comment (shobhitka) [inline groups/android_ia/default/init.rc]: The above changes are unrelated to the uevent issue and should go in a separate patch.

**Comment by shobhitka** [comment]
@sgnanase , can  you please test and update results here. Especially check for all modules being loaded properly and all devices like wifi, BT, etc are functional.

File: groups/android_ia/default/init.rc (hunk 2 above)
Comment (tanminger) [inline groups/android_ia/default/init.rc]: I create a new pull requests for it: https://github.com/android-ia/device-androidia-mixins/pull/86

**Comment by tanminger** [comment]
According @shobhitka feedback, reorg the patch sets, and create a new pull request https://github.com/android-ia/device-androidia-mixins/pull/86.
